common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -195,10 +206,12 @@ object ContentWidths {
       wide =            Some(1920.px))
   }
 
-  def getWidthsFromContentElement(hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
+  def getWidthsFromContentElement(contentType: String, hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
     relation match {
       case MainMedia => hinting.mainContentWidths
-      case _ => hinting.bodyContentWidths }
+      case BodyMedia if hinting == ContentWidths.Inline && contentType == "LiveBlog" => LiveBlogMedia.Inline
```

Comment:
this feels messy, why do we need a special if just for live blog, but not for other content types?